woocommerce-admin-test-helper.php

```diff
@@ -50,10 +50,14 @@ function add_extension_register_script() {
 		),
 		$css_file_version
 	);
+
 	wp_enqueue_style( 'woocommerce-admin-test-helper' );
+
+    wp_enqueue_style('woocommerce-styles', '/wp-content/plugins/woocommerce-admin/dist/components/style.css');
```

Comment:
This won't work if the plugin isn't installed (if it is just running against the bundled version within WC core).

Reply:
That's my mistake. I included it temporarily when I was testing WC Admin component. I will take it out.